### ISSUE TITLE
Nuchal edema

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5857]
**Preferred term label:**
Nuchal edema

**Synonyms**

**Definition (free text, please give PubMed ID)**
Nuchal edema is considered present if, in the midsagittal plane of the neck, there is subcutaneous edema that produces a characteristic tremor on ballotment of the fetal head. This constitutes the severe end of the spectrum of increased nuchal fold thickness, which is defined as soft-tissue thickening of 6 mm or more, seen in the suboccipitobregmatic view of the fetal head.3 Nuchal edema may be confined to the neck or it may be generalized, as part of hydrops fetalis. PMID: 16507941

Comment: In the fetus fluid collects behind the neck, much like it does in dependent ankle oedema in later life. This occurs partly because of the fetus’s tendency to lie on its back and partly because of the laxity of the skin of the neck. As with ankle oedema this accumulation of fluid can represent the end point of several pathological processes, including heart failure. Fluid collecting behind the neck can be detected as nuchal translucency by ultrasound scanning, and it can be measured. The more fluid that has accumulated, the greater the risk of an abnormality being present. PMID:9880279

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
see https://pubmed.ncbi.nlm.nih.gov/31130282/, 3/6

**Comment by pnrobinson** [comment]
What should the relationship be to existing related terms such as https://hpo.jax.org/app/browse/term/HP:0010880

**Comment by patt1morgan** [comment]
@sparkste, @jokemuys.   Thank you for agreeing to help us advance the perinatal section of the HPO. We would appreciate it if you could review this new term request and approve it or help us improve it. Any suggestions welcome!

**Comment by SophieCollardeau** [comment]
increased nuchal translucency could be a synonym

**Comment by pnrobinson** [comment]
@SophieCollardeau @sparkste Should "Nuchal edema" with the above definition be a synonym for 
[Increased nuchal translucency HP:0010880](https://hpo.jax.org/app/browse/term/HP:0010880)?

**Comment by pnrobinson** [comment]
Fetal neck anomaly 
-- Increased nuchal translucency (only GW 11-14 -- revise definition!)
-- Nuchal edema (Better Increased NF thickness) -- after GW14
-- Fetal cystic hygroma

**Comment by SophieCollardeau** [comment]
Increased NT refers to a measurement above the 95th centile, and the term is used irrespective of whether the collection of fluid is septated or not, and whether it is confined to the neck or envelopes the whole fetus.[1](https://www.sciencedirect.com/science/article/pii/S0002937805001183#bib1) After 14 weeks, increased NT usually resolves, but in some cases it evolves into nuchal edema or cystic hygromas (https://www.sciencedirect.com/topics/medicine-and-dentistry/cystic-hygroma).[3](https://www.sciencedirect.com/science/article/pii/S0002937805001183#bib3), [4](https://www.sciencedirect.com/science/article/pii/S0002937805001183#bib4), [5](https://www.sciencedirect.com/science/article/pii/S0002937805001183#bib5), [6](https://www.sciencedirect.com/science/article/pii/S0002937805001183#bib6) 
Increased nuchal translucency is a radiological  term   introduced in 1992  ( K.H. Nicolaides, G. Azar, D. Byrne, C. Mansur, K. Marks Fetal nuchal translucency: ultrasound screening for chromosomal defects in first trimester of pregnancy
BMJ, 304 (1992), pp. 867-889) corresponding to the ultrasonic feature of nuchal oedema or cystic hygromas.
On gross examination of fetuses we use the term nuchal oedema or nuchal thickness
and the term cystic hygroma when large cystic  cavities are seen

**Comment by SophieCollardeau** [comment]
https://doi.org/10.1016/j.ajog.2004.12.093.

**Comment by SophieCollardeau** [comment]
yes we should  keep these  three terms
-Fetal nuchal edema
-Cystic hygroma
-Thickened nuchal skin fold

this last one is indeed obseved in  syndromes  (Zellweger for example) or sequence (such as Potter 's facial dysmorphism) different from those which present with fetal nuchal oedema

**Comment by pnrobinson** [comment]
done, thanks!
